test/e2e/tests/tokens/import-tokens.spec.ts

```diff
@@ -90,7 +90,10 @@ describe('Import flow', function () {
         ]);
 
         const tokenList = new AssetListPage(driver);
-        await tokenList.check_tokenItemNumber(5); // Linea & Mainnet Eth
+
+        // Native Tokens: Ethereum ETH, Linea ETH, Base ETH, Polygon POL
+        // ERC20 Tokens: Chain Games, Chai
+        await tokenList.check_tokenItemNumber(6);
```

Comment:
Updated e2e tests as popular network view also shows the Base ETH token.

Added comments to make this a bit more clear.

Reply:
I wonder if there is a nice way of making this test more robust by filtering out the native tokens we are checking?